Tasks/AzureNLBManagement/task.json

```diff
@@ -0,0 +1,120 @@
+{
+	"id": "e94f1750-a6a8-11e6-be69-bdf37a7b15d8",
+	"name": "AzureNLBManagement",
+	"friendlyName": "Azure Network Load Balancer (Preview)",
+	"description": "Connect/Disconnect an Azure virtual machine's network interface to a Load Balancer's backend address pool",
+	"author": "Microsoft Corporation",
+	"helpMarkDown": "Replace with markdown to show in help",
+	"category": "Utility",
+	"visibility": [
+		"Build",
+		"Release", 
+        "Preview"
+    ],
+    "runsOn": [
+        "MachineGroup"
+    ],
+	"version": {
+		"Major": "1",
+		"Minor": "0",
+		"Patch": "0"
+	},
+	"minimumAgentVersion": "1.95.0",
+	"instanceNameFormat": "Azure Network Load Balancer: $(Action) VMs from $(LoadBalancer)",
+	"inputs": [
+		{
+            "name": "ConnectedServiceName",
+            "type": "connectedService:AzureRM",
+            "label": "AzureRM Subscription",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Select the Azure Resource Manager subscription for the deployment."
+		}, 
+		{
+            "name": "ResourceGroupName",
+            "type": "pickList",
+            "label": "Resource Group",
+            "required": true,
+            "helpMarkDown": "Select the resource group name.",
+            "properties": {
+                "EditableOptions": "True"
+            }
+        }, 
+        {
+            "name": "LoadBalancer",
+            "type": "pickList",
+            "label": "Load Balancer Name",
+            "required": true,
+            "helpMarkDown": "Select or enter the load balancer.",
+            "properties": {
+                "EditableOptions": "True"
+            }
+        }, 
+        {
+            "name": "Action",
+            "type": "pickList",
+            "label": "Action",
+            "defaultValue": "",
+            "required": true,
+            "options": {
+                "Disconnect": "Disconnect Network Interface",
+                "Connect": "Connect Network Interface"
+            },
+            "helpMarkDown": "Connect: Adds the virtual machine’s network interface to load balancer backend pool. So that it starts receiving network traffic based on the load balancing rules for the load balancer resource.\n\nDisconnect:  Removes the virtual machine’s network interface from the load balancer’s backend pool. So that it stops receiving network traffic."
+        },
+        {
+            "name": "NICDetection",
+            "type": "pickList",
+            "label": "NIC Detection",
+            "required": true,
+            "options": {
+                "AutoDetectNic": "Auto select primary network interface",
+                "CustomNic": "Custom - virtual machine : network interface"
```

Comment:
Is Custom just good enough. Do we need the mapping format in this drop down label?